assets/css/base/gutenberg-blocks.scss

```
@@ -306,6 +306,9 @@
 
 	// Image
 	.wp-block-image {
```

we should try to scope this to normal images only
```suggestion
	.wp-block-image:not( .block-editor-media-placeholder ) {
```